support/ccm.rb

```diff
@@ -600,7 +592,13 @@ def add_node(name)
 
       i = name.sub('node', '')
 
-      @ccm.exec('add', '-b', "-t 127.0.0.#{i}:9160", "-l 127.0.0.#{i}:7000", "--binary-itf=127.0.0.#{i}:9042", name)
+      if @dse
```

Comment:
If you construct an array of args and then invoke exec with *args, you can reduce duplication here.